File: Remove Duplicates from Sorted Array.py
class Solution:
	# @param a list of integers
	# @return an integer
	def removeDuplicates(self, A):
		if (len(A) == 0):
			return 0
		w = 0
		last = A[0] - 1
		for r in xrange(len(A)):
			if (A[r] != last):
				A[w] = A[r]
				last = A[r]
				w = w + 1
		return w
		

File: Remove Duplicates from Sorted Array.cpp
class Solution {
public:
    int removeDuplicates(int A[], int n) {
        if (n == 0)
            return 0;
        int w = 1;
        for (int r = 1; r < n; r++)
        {
            if (A[r] == A[w-1])
                continue;
            else
            {
                A[w++] = A[r];
            }
        }
        return w;
    }
};